languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -63936,6 +63936,63 @@ USA
             </rule>
         </rulegroup>
+        <rulegroup id="DOUBLE_NEG_RP" name="I didn't mention it to no one" default="temp_off">
+            <antipattern>
+                <token regexp="yes">could|would|should</token>
+                <token regexp="yes">not|n't</token>
+                <token postag="VB" />
+            </antipattern>
+            <rule>
+                <antipattern>
+                    <token regexp="yes">come|present</token>
+                    <token regexp="yes">from|in</token>
+                    <token>nothing</token>
+                </antipattern>
+                <pattern>
+                    <token regexp="yes">not|n't</token>
+                    <token postag="VB">
+                        <exception regexp="yes" inflected="yes">die|get|try</exception>
+                    </token>
+                    <token postag="IN|RP|TO" postag_regexp="yes" />
+                    <marker>
+                        <token>nothing</token>
+                    </marker>
+                </pattern>
+                <message>This phrase contains a double negative, or a comma may be missing. Try: <suggestion>anything</suggestion>.</message>
+                <example correction="anything">I won't stop for <marker>nothing</marker>.</example>
+                <example>From nothing there will always be something because a rule that “something can't come from nothing” would be something, and therefore it is not present in nothing.</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <token regexp="yes">not|n't</token>
+                    <token postag="VB" />
+                    <token min="0" chunk_re="B\-NP\-(singular|plural)" />
+                    <token min="0" chunk_re="E\-NP\-(singular|plural)" />
+                    <token postag="IN|RP|TO" postag_regexp="yes" />
+                    <marker>
+                        <token>nobody</token>
+                    </marker>
+                </pattern>
+                <message>This phrase contains a double negative, or a comma may be missing. Try: <suggestion>anybody</suggestion>.</message>
+                <example correction="anybody">I didn't show it to <marker>nobody</marker>.</example>
+            </rule>
+            <rule>
+                <pattern>
+                    <token regexp="yes">not|n't</token>
+                    <token postag="VB" />
+                    <token min="0" chunk_re="B\-NP\-(singular|plural)" />
+                    <token min="0" chunk_re="E\-NP\-(singular|plural)" />
+                    <token postag="IN|RP|TO" postag_regexp="yes" />
+                    <marker>
+                        <token>no</token>
+                        <token>one</token>
+                    </marker>
+                </pattern>
+                <message>This phrase contains a double negative, or a comma may be missing. Try: <suggestion>anyone</suggestion>.</message>
```

Comment:
We could hedge the message some more, like "might contain". Just in case somebody actually means to use litotes.

Reply:
Fixed 💯